File: week_6/cash/cash.py
from cs50 import get_float

# Get total $
dollars = 0
while (dollars <= 0):
    dollars = get_float("Change owed: ")

# Round and transfer to cents (Dollars To Cents)
cents = round(dollars * 100, 0)
coins = 0

# Calculating the number of coins
coins += cents // 25
cents = cents % 25

coins += cents // 10
cents = cents % 10

coins += cents // 5
cents = cents % 5

coins += cents // 1
cents = cents % 1

# displays the total number of coins (float To Int)
print(int(coins))

File: week_1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    int cents;
    float n;
    int total_coins = 0;
    int temp;
    int mod;

    do
    {
        n = get_float("Change owed: ");
    }
    while (n < 0);



    cents = round(n * 100);

    temp = cents / 25;
    mod = cents % 25;
    total_coins += temp;

    temp = mod / 10;
    mod = mod % 10;
    total_coins += temp;

    temp = mod / 5;
    mod = mod % 5;
    total_coins += temp;

    temp = mod / 1;
    mod = mod % 1;
    total_coins += temp;

    printf("%i \n", total_coins);
}